### PR TITLE
Add eight redirects for works

### LINKED ISSUE #2074: "doctoral thesis" (Q187685) does not redirect to work
**Describe the bug**
"doctoral thesis" (Q187685) does not redirect to work

**To Reproduce**
1. Go to https://scholia.toolforge.org/Q112659099
4. Redirected to https://scholia.toolforge.org/topic/Q112659099

**Expected behavior**
Redirected to https://scholia.toolforge.org/work/Q112659099

### PR DESCRIPTION
**Pull request by carlinmack**
Fixes #2074

    
Add redirects for doctoral thesis as well as thesis and masters thesis as well as five others.

Before merging it would be worth you also considering if these are valid

### Caveats

> Please list anything which has been left out of this PR or which should be considered before this PR is accepted
Check any of the following which apply:

* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ]  This change requires a documentation update
    * [ ]  I have made corresponding changes to the documentation
* [ ]  This change requires new dependencies (please list)

*If you make changes to the Python code*
  
* [ ]  My code passes the [tox](https://tox.readthedocs.io/en/latest/) check, I can receive warnings about tests, documentation or both

### Testing
Please describe the [tests](https://numpy.org/doc/stable/reference/testing.html) that you ran to verify your changes. Provide instructions, so we can reproduce. Please also list any relevant details for your test configuration.

* Test A
* Test B

### Checklist
* [ ] I have [commented](https://peps.python.org/pep-0008/#comments) my code, particularly in hard-to-understand areas
* [ ] My changes generate no new warnings
* [ ] I have not used code from external sources without attribution
* [ ] I have considered [accessibility](https://www.w3.org/standards/webdesign/accessibility) in my implementation 
* [ ] There are no remaining debug statements (print, console.log, ...)

### CODE REVIEW

**Comment by carlinmack** [comment]
I went through the [top 100 P31 by number of instances](https://w.wiki/5gUN) from scholia.toolforge.org/P31 and checked whether we have a redirect where applicable. I couldn't find any [case studies that weren't academic articles](https://w.wiki/5gUK
) so I didn't add a redirect for that

File: scholia/query.py [hunk 1]
```diff
@@ -1114,6 +1114,7 @@ def q_to_class(q):
             'Q571',  # book
             'Q49848',  # document
             'Q191067',  # article
+            'Q871232'  # editorial
```
Comment (fnielsen) [inline scholia/query.py:1117]: Missing comma

File: scholia/query.py [hunk 2]
```diff
@@ -1122,7 +1123,9 @@ def q_to_class(q):
             'Q5707594',  # news article
             'Q10870555',  # report
             'Q10885494',  # scientific conference paper
-            'Q13442814',  # scientific article
+            'Q4119870'  # academic writing
```
Comment (fnielsen) [inline scholia/query.py:1126]: Missing comma

File: scholia/query.py [hunk 3]
```diff
@@ -1133,6 +1136,11 @@ def q_to_class(q):
             'Q56119332',  # tweet
             'Q58632367',  # conference abstract
             'Q64548048',  # environmental impact assessment report
+            'Q1266946',  # thesis
+            'Q1907875',  # master's thesis
+            'Q187685',  # doctoral thesis
+            'Q815382'  # meta-analysis
```
Comment (fnielsen) [inline scholia/query.py:1142]: Missing comma

File: scholia/query.py [hunk 4]
```diff
@@ -1133,6 +1136,11 @@ def q_to_class(q):
...
+            'Q1778788'  # cohort study
```
Comment (fnielsen) [inline scholia/query.py:1143]: Missing comma

**Pull request merged**